brooklyn-tosca-transformer/src/test/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaTypePlanTransformerIntegrationTest.java

```diff
@@ -189,6 +182,51 @@ public void testRelation() throws ParsingException, CSARVersionAlreadyExistsExce
                 "echo This is the target");
     }
 
+
+    @Test
+    public void testRelationNotOverridePropCollectionConfigKey()
```

Comment:
These tests are good but in general prefer adding unit tests with mocked dependencies because they're _much_ quicker to run repeatedly than these integration tests that require a full Spring context.